src/App.tsx

```diff
@@ -1,14 +1,82 @@
+/* eslint-disable react-hooks/rules-of-hooks */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { Loader } from './components/Loader';
+import { TodoFilter, FilterTypes } from './components/TodoFilter';
+import { Todos } from './components/Todos';
 
-const USER_ID = 0;
+import { client } from './utils/fetchClient';
+
+import { Todo } from './types/Todo';
+
+const USER_ID = '10682';
 
 export const App: React.FC = () => {
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [createTodo, setCreateTodo] = useState<string>('');
+  const [filter, setFilter] = useState(FilterTypes.All);
+  const [isLoader, setIsLoader] = useState<boolean>(true);
+
+  useEffect(() => {
+    setIsLoader(true);
+
+    client.get(USER_ID);
```

Comment:
create a separate function (in api.ts file) which uses `client.get` and provide correct URL (check arguments which `get` expects)